packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -18,14 +18,18 @@ import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixi
 
 const multiSelectComboBox = css`
   :host {
-    --chip-min-width: var(--vaadin-multi-select-combo-box-chip-min-width, 4em);
     --input-min-width: var(--vaadin-multi-select-combo-box-input-min-width, 4em);
   }
 
   [hidden] {
     display: none !important;
   }
 
+  #chips {
```

Comment:
I'm not sure if we should add `part` attribute to this chips wrapper?

It would probably make sense if we want to follow this suggestion:

> refactor the chips to use CSS gap for spacing, and maybe also so that the overflow chip would be included in the same container as the other chips

Reply:
Part makes sense. Someone may want to change the spacing.

Reply:
Added the part.